engine/core/src/main/java/org/datacleaner/util/sort/SortMergeWriter.java

```diff
@@ -45,8 +46,6 @@
  * Sorter, deduplicator and writer that uses temporary files as storage to
  * support high volume sorted data.
  * 
- * Note: This class is NOT thread-safe.
```

Comment:
You're right ... Another silly thing happened (feedback from our tester Jan Arie) and I accidentally committed the fix on this branch ... DOH.